unicodetools/src/main/java/org/unicode/text/UCA/UCA.java

```diff
@@ -1799,61 +1809,4 @@ public static UCA buildCollator(Remap primaryRemap) {
     UCA_Statistics getStatistics() {
         return ucaData.statistics;
     }
-
-    public static final class Remap {
-        private int counter = 0x100;
-        private final Map<Integer, Integer> primaryRemap = new TreeMap<Integer, Integer>();
-        private final Map<Integer, IntStack> characterRemap = new TreeMap<Integer, IntStack>();
-        private int variableHigh;
-        private int firstDucetNonVariable;
```

Comment:
FYI: We still need to carry these. The code now just moves these two primaries around rather than the otherwise obsolete Remap object.